tests/ember_debug/view-debug-test.js

```diff
@@ -131,7 +131,6 @@ test("Simple View Tree", async function t(assert) {
   let value = tree.value;
   assert.equal(tree.children.length, 1);
   assert.equal(value.controller.name, 'App.ApplicationController');
-  assert.equal(value.viewClass, '(unknown mixin)');
```

Comment:
In glimmer seems that there is no views anymore. I don't know if it's intended or an oversight. Can anyone confirm?